sdk/python/kfp/dsl/component_decorator.py

```diff
@@ -75,6 +76,9 @@ def component(func: Optional[Callable] = None,
             as that used when this component was created. Component authors can
             choose to override this to point to a GitHub pull request or
             other pip-compatible package server.
+        use_venv: Specifies if the component should be executed in a virtual environment.
+        The environment will be created in a temporary directory and will inherit the system site packages.
+        This is useful in restricted environments where most of the system is read-only.
```

Comment:
nit: can you indent the 2nd/3rd lines here to match the formatting of the previous parameter docstrings

Reply:
Ack!